context_chat_backend/vectordb/pgvector.py

```diff
@@ -0,0 +1,149 @@
+from logging import error as log_error
+from typing import Any
+
+import sqlalchemy as sa
+from dotenv import load_dotenv
+from langchain.vectorstores import VectorStore
+from langchain_core.embeddings import Embeddings
+from langchain_postgres import PGVector
+
+from . import get_collection_name, get_user_id_from_collection
+from .base import BaseVectorDB, DbException, MetadataFilter, TSearchDict
+
+load_dotenv()
+
+
+class JSONB(sa.sql.sqltypes.Indexable, sa.sql.sqltypes.TypeEngine[Any]):  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
Why can't we use the JSONB type directly from SqlAlchemy?

Reply:
It doesn't exist for some reason, only `JSON`